db/re/skill_require_db.txt

```diff
@@ -916,27 +920,27 @@
-2590,0,0,40,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0								//SJ_LIGHTOFSUN##
-2591,0,0,10,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0								//SJ_SUNSTANCE##
-2592,0,0,34:37:40:43:46:49:52,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0			//SJ_SOLARBURST##
-2593,0,0,20,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0								//SJ_PROMINENCEKICK##
-2594,0,0,1,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0								//SJ_FALLINGSTAR_ATK##
-2595,0,0,1,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0								//SJ_FALLINGSTAR_ATK2##
+2574,0,0,40,0,0,0,99,0,0,moonstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 //SJ_LIGHTOFMOON##
+2575,0,0,10,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 //SJ_LUNARSTANCE##
+2576,0,0,30:35:40:45:50:55:60,0,0,0,99,0,0,moonstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_FULLMOONKICK##
+2577,0,0,40,0,0,0,99,0,0,starstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_LIGHTOFSTAR##
+2578,0,0,10,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 //SJ_STARSTANCE##
+2579,0,0,20:25:30:35:40:45:50,0,0,0,99,0,0,moonstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 //SJ_NEWMOONKICK##
+2580,0,0,45:40:35:30:25:20:15,0,0,0,99,0,0,starstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_FLASHKICK##
+2581,0,0,70:75:80:85:90,0,0,0,99,0,0,universestance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_STAREMPEROR##
+2582,0,0,60:65:70:75:80,0,0,0,99,0,0,universestance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_NOVAEXPLOSING##
+2583,0,0,10,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_UNIVERSESTANCE##
+2584,0,0,40:45:50:55:60:65:70,0,0,0,99,0,0,starstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_FALLINGSTAR##
+2585,0,0,80,0,0,0,99,0,0,universestance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_GRAVITYCONTROL##
+2586,0,0,40,0,0,0,99,0,0,universestance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_BOOKOFDIMENSION##
+2587,0,0,50:55:60:65:70,0,0,0,99,0,0,universestance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_BOOKOFCREATINGSTAR##
+2588,0,0,60,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_DOCUMENT##
+2590,0,0,40,0,0,0,99,0,0,sunstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 //SJ_LIGHTOFSUN##
+2591,0,0,10,0,0,0,99,0,0,none,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_SUNSTANCE##
+2592,0,0,34:37:40:43:46:49:52,0,0,0,99,0,0,sunstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 //SJ_SOLARBURST##
+2593,0,0,20,0,0,0,99,0,0,sunstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0 //SJ_PROMINENCEKICK##
+2594,0,0,1,0,0,0,99,0,0,starstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_FALLINGSTAR_ATK##
+2595,0,0,1,0,0,0,99,0,0,starstance,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0	//SJ_FALLINGSTAR_ATK2##
```

Comment:
these could have just added SC_STARSTANCE (and so on) i know a special exception has to be made, placing them into skill_check_condition_sc_required. i dunno which is best. since the code on either side can handle these.